lib/global-admin/addon/components/cru-cloud-credential/component.js

```diff
@@ -380,9 +382,11 @@ export default Component.extend(ViewNewEdit, {
   },
 
   setError(message = '') {
+    set(this, 'cloudErrorExists', true);
     const translation = this.intl.t('modalAddCloudKey.errors.validation', { status: message });
 
     set(this, 'validatingKeys', false);
+    set(this, 'cloudError', this.intl.t('modalAddCloudKey.errors.errorMessage'));
```

Comment:
this `setError` function is supposed to be a generic error setting function. I think we should be not modifying this function (only calling it with the correct error message on L360)